tests/test_graph.py

```diff
@@ -1,8 +1,37 @@
+import pytest
 from paracelsus.graph import get_graph_string
 
 from .utils import mermaid_assert
 
 
 def test_get_graph_string(package_path):
-    graph_string = get_graph_string("example.base:Base", ["example.models"], [package_path], "mermaid")
+    graph_string = get_graph_string("example.base:Base", ["example.models"], set(), set(), [package_path], "mermaid")
```

Comment:
Since this is starting to get really long I'd prefer if we switched to named arguments rather than positional arguements. ruff/black will then format them so one arg is on each line.

Reply:
Done